[EICNET-1774]feat: Add tagging user system on comment

Dromadoros opened the pull request:
How to test it:

- [x] Go to a discussion detail
- [x] Add a comment and you will see all possible users depending the visibility of the group
- [x] Select one/multiple user(s)
- [x] send comment
- [x] You should see your new comments with a new line into the comment box "With: XXX"

lramarojaona commented on lib/modules/eic_groups/src/Controller/DiscussionController.php:
```diff
@@ -135,6 +137,11 @@ public function addComment(Request $request, $discussion_id) {
         'value' => $text,
         'format' => 'plain_text',
       ],
+      'field_tagged_users' => array_map(function($tagged_user) {
```
This variable isn't set anywhere `$tagged_user`. I guess this should be `$tagged_users`?

Dromadoros replied:
It's an array_map so it's coming from tagged_users (second param of array_map)

lramarojaona commented on lib/themes/eic_community/react/components/Block/CommentsDiscussion/Partials/CommentForm.js:
```diff
@@ -1,64 +1,167 @@
+                        <button className="close" onClick={close}>
+                          &times;
+                        </button>
+                        <div className="header">{"test"}</div>
```
This shouldn't be here I guess

lramarojaona commented on lib/themes/eic_community/react/components/Block/CommentsDiscussion/Partials/CommentForm.js:
```diff
@@ -1,64 +1,167 @@
+                                <div className="ecl-author ecl-author--is-default  ecl-author--hide-info">
+                                  <div className="ecl-author__information">
+                              <span className="ecl-author__label ">
+                                    <a href="?path=user-john-doe"
```
Do we need to show an author here?

lramarojaona reviewed (changes requested):
It works althoug the modal window doesn't seem complete to me (it's showing a test message + showing an author, is this needed?)

I added a comment in the ticket regarding emails, cause to me it seems useless if tagged users don't get an email, but it's maybe not in the scope of this ticket

Dromadoros commented:
> It works althoug the modal window doesn't seem complete to me (it's showing a test message + showing an author, is this needed?)
> 
> I added a comment in the ticket regarding emails, cause to me it seems useless if tagged users don't get an email, but it's maybe not in the scope of this ticket

I removed the author + test title. And I agree, we can see the email point in a discussion/ticket
